src/main/ipc.ts

```diff
@@ -0,0 +1,33 @@
+import { ipcMain } from 'electron';
+import Procedures from '../db/Procedures';
+import ConnectionService from '../db/service/ConnectionService';
+
+ipcMain.on('ipc-example', async (event, arg) => {
+  const msgTemplate = (pingPong: string) => `IPC test: ${pingPong}`;
+  console.log(msgTemplate(arg));
```

Comment:
Though not necessary for this PR, we should look into using a logger than just basic `console.log`

Reply:
Got it, will add this as a ticket!